### PR TITLE
adds missing quotes on envsubst strings

### PR DESCRIPTION
**Pull request by butlerpaul**
Signed-off-by: Paul Butler <butler.paul@gmail.com>

### CODE REVIEW

**Review by KeilChris — COMMENTED** [review]
Hello Paul, why have you added this file for CI/CD usage? The AMI is not using WiFi, so this socket_startup.c file should not be necessary. @RobertRostohar, can you please comment?

File: amazon-freertos/demos/include/aws_clientcredential_keys.h.in [hunk 1]
```diff
@@ -52,7 +52,7 @@
  */
 #ifndef keyCLIENT_CERTIFICATE_PEM
     #define keyCLIENT_CERTIFICATE_PEM    \
-    ${CLIENT_CERTIFICATE_PEM}
+    "${CLIENT_CERTIFICATE_PEM}"
```
Comment (RobertRostohar) [inline amazon-freertos/demos/include/aws_clientcredential_keys.h.in]: The quotes are not needed if we agree to enter the secret in the following format (as shown a few lines above in the comments) - without the trailing back-slash:
"-----BEGIN CERTIFICATE-----\n"
"...base64 data...\n"
"-----END CERTIFICATE-----\n"

Instructions for converting the PEM file is just add " at the beginning and \n" at the end of each line.

File: amazon-freertos/demos/include/aws_clientcredential_keys.h.in [hunk 2]
```diff
@@ -97,7 +97,7 @@
  */
 #ifndef keyCLIENT_PRIVATE_KEY_PEM
     #define keyCLIENT_PRIVATE_KEY_PEM    \
-    ${CLIENT_PRIVATE_KEY_PEM}
+    "${CLIENT_PRIVATE_KEY_PEM}"
```
Comment (RobertRostohar) [inline amazon-freertos/demos/include/aws_clientcredential_keys.h.in]: See previous comment.

**Review by RobertRostohar — CHANGES_REQUESTED** [review]
socket_startup.c.in should not be needed since CD/CI doesn't use WiFi.

File: amazon-freertos/demos/include/aws_clientcredential_keys.h.in (hunk 1 above)
Comment (butlerpaul) [reply amazon-freertos/demos/include/aws_clientcredential_keys.h.in]: In the workshop I provide a script to convert the strings and apply the changes in the code for them. Manually editing the certs to include the quotes and newlines is tedious. I update socket_startup.c as well to quickly switch between AVH and U5 build. socket_startup.c in only needed for the U5 build, but if it is ready with SSID and PASSWORD, all you have to do is switch the target and rebuild. (Correct, this is not really used in the CI/CD), but helps with KSC convenience.

**Comment by KeilChris** [comment]
Paul, with your PR, the GH action breaks. I'm not sure why. Can you please check?

**Comment by butlerpaul** [comment]
Checking now

-paul

Paul Butler
***@***.***
Mobile: 214-250-6159

File: amazon-freertos/demos/include/aws_clientcredential_keys.h.in (hunk 1 above)
Comment (butlerpaul) [reply amazon-freertos/demos/include/aws_clientcredential_keys.h.in]: I'll just modify my script for usage in KSC. Then this PR will not be needed.

**Pull request closed without merging**